frontend/src/v5/services/hotjar.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import Hotjar from '@hotjar/browser';
+import { clientConfigService } from '../../v4/services/clientConfig';
+
+export const initializeHotjar = () => {
+	const { development, hotjar } = clientConfigService;
+
+	if (!development && hotjar) {
+		const { siteId, version } = hotjar;
+		Hotjar.init(siteId, version);
```

Comment:
@Amantini1997 @carmenfan 
I tested this branch and even though I see that the hotjar script is downloading:
<img width="1016" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/320b8b08-ef70-4724-9fae-fba0ebdb6a72">

I don't see any results in the website ( I waited almost an hour)
<img width="943" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/3b091387-8797-45f2-8c55-cd81d9a73a0c">

I also don't see any request being done to hotjar besides downloading the script

Reply:
It looks like `clientConfig` is sending `hotjar` as `undefined`:
![image](https://github.com/3drepo/3drepo.io/assets/43547703/8106e574-ab57-4fb5-8584-6d0c15793b6b)
For the records, I don't remember how long I had to wait to see the results the first time, but I think I had to wait until the next day

Reply:
The deployed branch is not configured correctly. I was talking about testing it in your local version

Reply:
Same in local, `hotjar` seems to be missing 
![image](https://github.com/3drepo/3drepo.io/assets/43547703/01922009-717c-4593-b6af-de9aa756901d)

Reply:
You didnt configured local correctly . check a look at https://github.com/3drepo/3drepo.io/pull/4618#issuecomment-1889310432

Reply:
deployed branch won't be configured until we get devops to set it up. so only test on local please 😆

Reply:
It took several more hours than I expected but got to see the results